__tests__/.env.e2e.example

```
@@ -0,0 +1,92 @@
+# INFRASTRUCTURE - Required for all E2E tests
+# ==============================================================================
+
+# Redis connection URL
+# Use port 16379 (from docker-compose.e2e-test.yml) to avoid conflict with dev Redis
+REDIS_URL=redis://localhost:16379
+
+# Neo4j connection settings
+# Use port 17687 (from docker-compose.e2e-test.yml) to avoid conflict with dev Neo4j
+NEO4J_URI=bolt://localhost:17687
+NEO4J_USER=neo4j
+NEO4J_PASSWORD=e2e-test-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEO4J_PASSWORD key should go before the NEO4J_URI key

(UnorderedKey)